lib/Jsrt/JsrtInternal.h

```diff
@@ -400,7 +400,7 @@ JsErrorCode SetContextAPIWrapper(JsrtContext* newContext, Fn fn)
 void HandleScriptCompileError(Js::ScriptContext * scriptContext, CompileScriptException * se, const WCHAR * sourceUrl = nullptr);
 
 #if DBG
```

Comment:
Does `DBG` require that `EXCEPTION_CHECK` is also defined?

Reply:
Right. After moving DebugCheckNoException under EXCEPTION_CHECK this will become a build break if DBG doesn't include EXCEPTION_CHECK (which I don't see why it would). Not a concern anymore as I'm reverting this part of the change.

---
In reply to: [196640917](https://github.com/Microsoft/ChakraCore/pull/5335#discussion_r196640917) [](ancestors = 196640917)